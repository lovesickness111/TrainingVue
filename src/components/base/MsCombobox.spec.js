import { mount } from "@vue/test-utils";
import MsCombobox from "src/components/base/MsCombobox.vue";
// import { describe, it, expect } from "vitest";

describe("MsCombobox.vue", () => {

  it("should renders is page content is correct", () => {
    const wrapper = mount(MsCombobox, {
      props: { name:"nvcuong1" },
    });

    expect(wrapper.vm.options).toMatchSnapshot();
  });
})