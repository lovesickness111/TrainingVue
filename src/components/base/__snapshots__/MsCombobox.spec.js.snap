// Vitest Snapshot v1

exports[`MsCombobox.vue > should renders is page content is correct 1`] = `[]`;
